karma.conf.js

```diff
@@ -181,7 +181,7 @@ function createEsbuildPlugin() {
 							coverage && [
 								'istanbul',
 								{
-									include: minify ? '**/dist/**/*.js' : '**/src/**/*.js'
+									include: minify ? '**/dist/**/*.{js,mjs}' : '**/src/**/*.js'
```

Comment:
Can we add `mjs` to the size tracker

Reply:
Compressed Size? Will do

Reply:
That actually needs to be addressed in the action itself.

It's _meant to_ support `.mjs` as-is, but we're using a [JS fallback](https://github.com/preactjs/compressed-size-action/blob/6fa0e7ca017120c754863b31123c5ee2860fd434/src/index.js#L45) when we define a default in the [action config](https://github.com/preactjs/compressed-size-action/blob/6fa0e7ca017120c754863b31123c5ee2860fd434/action.yml#L39). I'll write up a PR fixing that shortly, up to you on whether we want to hold off on this until that gets in though.

Reply:
All good now